hil-test/src/lib.rs

```diff
@@ -29,19 +29,19 @@ macro_rules! i2c_pins {
     ($io:expr) => {{
         cfg_if::cfg_if! {
             if #[cfg(any(esp32s2, esp32s3))] {
-                // For ESP32-S2 and ESP32-S3, use GPIO 2 and 3 for I2C
+                // For ESP32-S2 and ESP32-S3, use GPIO 2(SDA) and 3(SCL) for I2C
                 ($io.pins.gpio2, $io.pins.gpio3)
             } else if #[cfg(esp32c6)] {
-                // For ESP32-C6, use GPIO 6 and 7 for I2C
+                // For ESP32-C6, use GPIO 6(SDA) and 7(SCL) for I2C
                 ($io.pins.gpio6, $io.pins.gpio7)
             } else if #[cfg(esp32h2)] {
-                // For ESP32-H2, use GPIO 6 and 7 for I2C
-                ($io.pins.gpio4, $io.pins.gpio7)
-            } else if #[cfg(any(esp32c2, esp32c3))] {
-                // For ESP32-C2 and ESP32-C3, use GPIO 4 and 7 for I2C
-                ($io.pins.gpio4, $io.pins.gpio7)
+                // For ESP32-H2, use GPIO 4(SDA) and 22(SCL) for I2C
```

Comment:
I don't think you want to maintain these comments 🫠 Just remove them, there's no reason to duplicate the info.